concent_api/concent_api/database_router.py

```diff
@@ -0,0 +1,36 @@
+from concent_api.constants import APP_LABEL_TO_DATABASE
+
+
+class DatabaseRouter:
+    """ A router to control all database operations on models in Concent. """
+
+    def db_for_read(self, model, **hints):  # pylint: disable=unused-argument,no-self-use
```

Comment:
This is actually assert + one line which doesn't do any magic, so I don't see any sense in making this a separate function.

Reply:
and yet it is duplicated in another function;) Why would you allow Code Duplication in production code? :)

Reply:
assert and getting an item from dictionary are actually two lines. I am not going to argue about that, so if you don't like to change it leave it as it is ;)

Reply:
Yeah but asserts are not run in production mode. I just think it is too simple line to make a method out of it.

Reply:
I agree with @rwrzesien that extracting it because of an `assert` seems like an overkill.

And I really do not want to discourage from adding more assertions. They're cheap, easy and make your implicit assumptions explicit so we should really have more of them.